### PR TITLE
feature/CD-72: Styling dashboard images

### PR DESCRIPTION
**Pull request by gusartus**

### CODE REVIEW

File: src/assets/images/block-sign.svg [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Capa_1" x="0px" y="0px" width="512px" height="512px" viewBox="0 0 510 510" style="enable-background:new 0 0 510 510;" xml:space="preserve">
+<g>
+	<g id="block">
+		<path d="M255,0C114.75,0,0,114.75,0,255s114.75,255,255,255s255-114.75,255-255S395.25,0,255,0z M51,255c0-112.2,91.8-204,204-204    c45.9,0,89.25,15.3,124.95,43.35l-285.6,285.6C66.3,344.25,51,300.9,51,255z M255,459c-45.9,0-89.25-15.3-124.95-43.35    L415.65,130.05C443.7,165.75,459,209.1,459,255C459,367.2,367.2,459,255,459z" fill="#FFFFFF"/>
+	</g>
+</g>
+<g>
```
Comment (Gerasimov94) [inline src/assets/images/block-sign.svg]: maybe that <g> is unnecessary

File: src/assets/images/check.svg [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (Gerasimov94) [inline src/assets/images/check.svg]: same

File: src/modules/Dashboard/components/ImagesCardView/ImageCard/Controls/controls.scss [hunk 3]
```diff
@@ -3,37 +3,32 @@
...
 
 .control {
+	line-height: 13px;
 	display: flex;
 	flex-direction: column;
 	align-items: center;
-	justify-content: center;
-	font-size: 12;
+	justify-content: space-between;
+	height: 37px;
```
Comment (Gerasimov94) [inline src/modules/Dashboard/components/ImagesCardView/ImageCard/Controls/controls.scss]: why you are using a non-multiplied values? Eugene says our multiplier is 4, as i remember.

File: src/modules/Dashboard/components/ImagesCardView/ImageCard/ImageCard.tsx [hunk 4]
```diff
@@ -71,17 +77,13 @@ export default class ImageCard extends Component<IProps> {
 	private readonly errorStateControls: IControls[] = [
 		{
 			icon: (
-				<div className={controlsStyles.reactImg}>
-					<FaCircleNotch size={16} fill={COLORS.black} />
-				</div>
+				<FaCircleNotch size={19} fill={COLORS.black} />
```
Comment (Gerasimov94) [inline src/modules/Dashboard/components/ImagesCardView/ImageCard/ImageCard.tsx]: ? why 19? not 20? or 18? in my oppinion, all sizes must be like X mod 2 === 0

**Review by Gerasimov94 — CHANGES_REQUESTED** [review]

**Review by CodeAnnihilator — APPROVED** [review]

**Review by Gerasimov94 — APPROVED** [review]

**Pull request merged**